docs/input.rst

```diff
@@ -26,9 +26,35 @@ the *subscriber* named *MySubscriber*::
      <data_reader name="MySquareReader" topic_ref="Square" />
    </subscriber>
 
-This *publisher* is defined inside the *domain_participant* selected to create
+This *subscriber* is defined inside the *domain_participant* selected to create
 this ``connector`` (see :ref:`Create a new *Connector*`).
 
+Matching with an Output
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think sphinx will complain if the ~~~ don't match the number of characters of the title.